packages/cli/package.json

```diff
@@ -37,6 +37,7 @@
     "chalk": "^4.1.2",
     "commander": "^9.4.0",
     "compare-versions": "^5.0.1",
+    "cosmiconfig": "^8.0.0",
```

Comment:
~How is this dependency being used? Was it not added by accident?~
Saw @lynchbomb 's resolved comment.